harvard-lts/listview
# Added ImageMagick to fix missing image problem

**Image bug fix (no images allowed since containerizeing)*
* * *

**GitHub ticket: https://github.com/harvard-lts/HDC/issues/202

# What does this Pull Request do?
Fixes a bug in which users could no longer upload images.  Adding ImageMagick into the Dockerfile corrected the problem.  
Minor additional changes:
Enables logging in the configs 
Fixes a bug in a Rails 5 update that altered the way the back url call was used.


# How should this be tested?

Login to Hollis Listview QA https://listview-docker-qa.lib.harvard.edu/users/sign_in
Add an image to an existing list and save
No errors should occur and you should now see your image.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No
